upgrade/scripts/upgrade/upload-ncn-images.sh

```diff
@@ -26,13 +26,22 @@
 set -u
 
 CSM_ARTI_DIR=${CSM_DISTDIR:-''}
+CSM_RELEASE=${CSM_RELEASE:-''}
 
 if [[ -z ${CSM_ARTI_DIR} ]]; then
     echo "CSM_DISTDIR environment variable needs to be set and exported. It should be set to the path \
 of the extracted CSM product release."
     exit 1
 fi
 
+# CSM_ARTI_DIR and CSM_RELEASE are required for ncn-ims-image-upload.sh script used below
+export CSM_ARTI_DIR
```

Comment:
Does CSM_RELEASE need to be exported, too?

Reply:
Nevermind.  I see now that it is expected to be exported and doesn't get reassigned.